src/components/icons/ItalicTextIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import Icon from '../../@types/Icon'
+
+// eslint-disable-next-line jsdoc/require-jsdoc
+const ItalicTextIcon = ({ style, size }: Icon) => {
```

Comment:
Let's remove the border on this icon.

Reply:
I've changed Icon. The border was removed and _I_ is bigger.